src/tellor_disputables/data.py

```diff
@@ -85,24 +84,29 @@ async def log_loop(web3: Web3, addr: str) -> list[tuple[int, Any]]:
     num = web3.eth.get_block_number()
     try:
         events = web3.eth.get_logs({"fromBlock": num - 100, "toBlock": "latest", "address": addr})  # type: ignore
-    except ValueError as e:
-        if "unknown block" in e:
-            logger.log("waiting for new blocks")
-        elif "request failed or timed out" in e:
-            logger.log("request for eth event logs failed")
-        else:
-            logger.log("unknown RPC error gathering eth event logs \n" + e)
-    unique_events = {}
-    unique_events_lis = []
 
-    for event in events:
-        txhash = event["transactionHash"]
+        unique_events = {}
+        unique_events_list = []
 
-        if txhash not in unique_events:
-            unique_events[txhash] = event
-            unique_events_lis.append((web3.eth.chain_id, event))
+        for event in events:
+            txhash = event["transactionHash"]
 
-    return unique_events_lis
+            if txhash not in unique_events:
+                unique_events[txhash] = event
+                unique_events_list.append((web3.eth.chain_id, event))
+
+        return unique_events_list
+
+    except ValueError as e:
+        msg = str(e)
+        if "unknown block" in msg:
+            print("waiting for new blocks")
```

Comment:
Should probs set up logging to a files similar to `telliot_core`/telliot_feeds`, so @spuddyminer can link txt files in the bug issues

Reply:
Referenced #24